hw/formal/README.md

```diff
@@ -311,6 +311,19 @@ For example:
           $past(rv_plic.le[src_sel]) || $past(intr_src_i[src_sel]), clk_i, !rst_ni)
 ```
 
+## Implementation Guidelines
+The recommended guidelines for where to implement assertions are as follows:
+* Basic assertions should be implemented directly in the RTL file. These basic
+  functional assertions are often inserted by designers to act as a sanity
+  check.
+* Assertions used for the testbench to achieve verification goals should be
+  implemented under the `ip/hw/module_name/fpv/vip` folder. This FPV environment
+  can be automatically generated by the [`fpvgen.py` script](../../util/fpvgen/README.md).
+* Portable assertions written for common interfaces or submodules should also be
```

Comment:
in this case, should we move `tlul/rtl/tlul_assert.sv` to `tlul/fpv/vip/tlul_assert.sv`?